apis/goharbor.io/v1alpha3/harborcluster_webhook.go

```diff
@@ -59,7 +59,7 @@ var _ webhook.Validator = &HarborCluster{}
 func (hc *HarborCluster) ValidateCreate() error {
 	clog.Info("validate creation", "name", hc.Name, "namespace", hc.Namespace)
 
-	return hc.validate(nil)
+	return hc.validate(hc)
```

Comment:
Why we still need v1alpha3 webhook?
Wher is the v1beta1 webhook?

Reply:
added in v1beta1/harborcluster